server/models/interview.js

```diff
@@ -0,0 +1,39 @@
+let mongoose = require("mongoose");
+
+let Interview = new mongoose.Schema(
+  {
+    owner: {
+      type: mongoose.ObjectId,
+      required: true,
+      ref: "users",
+    },
+    guest: {
+      type: mongoose.ObjectId,
+      ref: "users",
+    },
+    date: {
+      type: Date,
+      required: true,
+    },
+    theme: {
+      type: String,
+    },
+    questions: {
+      type: [mongoose.ObjectId],
```

Comment:
You should add a ref to this property as well

Reply:
The Questions model has not been added to the database, adding a reference pointing to an unexisting collection won't generate errors?